app/services/xokul/kps/verification.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Xokul
+  module Kps
+    module Verification
+      module_function
+
+      # rubocop:disable Metrics/MethodLength
+      def id_card(**option)
+        Connection.request(
+          '/kps/verifications/id_cards', params: {
+            id_number: option.fetch(:id_number),
+            first_name: option.fetch(:first_name),
+            last_name: option.fetch(:last_name),
+            day_of_birth: option.fetch(:day_of_birth),
+            month_of_birth: option.fetch(:month_of_birth),
+            year_of_birth: option.fetch(:year_of_birth),
+            serial: option[:serial],
+            number: option[:number],
+            document_number: option[:document_number]
+          }
+        )[:status]
+      end
+      # rubocop:enable Metrics/MethodLength
+
+      def id_number(**option)
```

Comment:
Ben bu id_number metodundan `Yabancı Uyruklu Kimlik` numarası doğrulayabilir miyim?

Reply:
`id_card` olarak dogrulayabilirsin. aldigi parametreler bazinda `id_card` dogrulamasina daha yakin. `id_card` yabanci uyruklu, Turk uyruklu ve yeni kimlik karti sahibi Turk uyruklu kisileri dogruluyor. Dokumanda belirttim: https://github.com/omu/xokul/blob/dev/doc/services/kps.md#verifications

Reply:
O zaman `id_number`'ı kaldıralım. iki parametre eksik vereceğiz diye neden metod ekliyelim.

Reply:
`id_number` neden kaldiralim? ikisi farkli seyler. bir `id_card` kimlik karti dogrulamasi yapiyor. `id_number` ise basit bir kimlik nimarasi dogrulamasi. tek bir islev ikiye bolunmus degil. yabanci uyruklulari bir gruba sokmam gerekiyordu `id_card` grubuna koydum

Reply:
Eğer `id_number` kalması gerekiyorsa bu da YU numarasını doğrulayabilmeli. Çünkü fark ettin mi bilmiyorum ama anlam karmaşası oluşuyor.

Reply:
Tamamdir. Senin demek istedigini anladim. Zaten Turk uyruklu kisiyi dogurlayabiliyorum. `id_number` dogrulamasi yapmama gerek yok, bu yuzden kaldiralim diyorsun. Tekrar onay alayim. Eger ihtiyac yoksa kaldiriyorum metodu.

Reply:
Eğer diğer arkadaşlarda benimle hem fikirse kaldıralım. `id_card` metodu dediğine göre, kimlik numara doğrulamalarını her iki uyruk için de yapıyor.

Reply:
Şöyle özetleyeyim.  @huseyin kps'deki iki farklı endpoint'i client tarafına yansılamış.  Bu hatalı bir karar değil.  Ama client API'yi basit de tutmak gerekiyor, bu açıdan @isubas'a katılıyorum.  Şimdilik tek bir metod olsun, gelecekte ortaya çıkacak kod paternlerine göre diğerini ekleriz.